.travis.yml

```diff
@@ -51,7 +51,7 @@ matrix:
         - npm run postinstall -- --scope "@loopback/test-repository-mongodb" --include-dependencies
         - npm run build -- --scope "@loopback/test-repository-mongodb" --include-dependencies
         - cd acceptance/repository-mongodb && npm run mocha
-    - node_js: "10"
+    - node_js: "12"
```

Comment:
why are we changing "10" to "12" then?
The title says we need to change "8" to "12"

Reply:
We don't need to change anything yet. But I _want_ to upgrade to the latest LTS version.

Reply:
I reworked the commit message as follows:

```
chore: upgrade Node to 12.x for Travis jobs

With Node.js version 12 already in LTS mode, it's time to upgrade
our Travis jobs to use the latest & greatest & fastest! version of Node.

Also note that then version 8 is going end of life soon, it's time
to upgrade to a newer one.
```

LGTY now?